openapi.yml

```diff
@@ -0,0 +1,578 @@
+openapi: 3.0.0
+info:
+  description: Preservation Catalog HTTP API
+  version: 1.0.0
+  title: Preservation Catalog HTTP API
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+servers:
+  - url: 'https://preservation-catalog-{env}-01.stanford.edu'
+    description: Production service
+    variables:
+      env:
+        default: prod
+  - url: 'https://preservation-catalog-{env}-01.stanford.edu'
+    description: Staging service
+    variables:
+      env:
+        default: stage
+  - url: 'https://preservation-catalog-{env}-01.stanford.edu'
+    description: QA service
+    variables:
+      env:
+        default: qa
+tags:
+  - name: catalog
+    description: Add and update MOABs in the catalog
+  - name: objects
+    description: Preserved Objects
+paths:
+  /v1/catalog:
+    post:
+      tags:
+        - catalog
+      summary: Add a new MOAB to the catalog
+      responses:
+        '201':
+          description: Created new MOAB
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AuditResultsResponse'
+        '409':
+          description: Conflict - MOAB already exists
+        '406':
+          description: Invalid arguments passed
+        '500':
+          description: Internal server error
+      requestBody:
+        required: true
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                druid:
+                  $ref: '#/components/schemas/Druid'
```

Comment:
does this take `bb111bb2222` as well as `druid:bb111bb2222` ?  because prescat does ...

Reply:
Yes. I made the `druid:` prefix optional in this schema's regex: https://github.com/sul-dlss/preservation_catalog/pull/1389/files#diff-dc51a340f5f12b9af5a5462818e6eca1R424

Resolving.